src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -251,6 +249,13 @@ public record Task(
     List<TaskEventSummary> Events { get; set; } = new List<TaskEventSummary>();
     List<NodeAssignment> Nodes { get; set; } = new List<NodeAssignment>();
 }
+
+public record TaskEvent(
+    [PartitionKey] Guid TaskId,
```

Comment:
This should be `[PartitonKey, RowKey]`, here's an example: https://github.com/microsoft/onefuzz/blob/cb1d89fce67f2b6f96c4d9c45b19d3f11213eee7/src/ApiService/ApiService/OneFuzzTypes/Model.cs#L309

Reply:
The "ORM" requires both to be present to deserialize the entity properly from azure table. If you have a test deployment you can look at the `func{random_letters}` storage account in the `TaskEvent` table to see that the `PartitionKey` and `RowKey` columns have the exact same data.

Reply:
Thanks!